.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
you can remove this file it is not required It was created by the "live server" extension.

Reply:
yeah you can delete that file.

Reply:
>
made the changes